src/components/views/elements/AppTile.js

```diff
@@ -499,6 +500,13 @@ export default class AppTile extends React.Component {
         }
     }
 
+    _onPopoutWidgetClick(e) {
+        // Using Object.assign workaround as the following opens in a new window instead of a new tab.
+        // window.open(this._getSafeUrl(), '_blank', 'noopener=yes,noreferrer=yes');
+        Object.assign(document.createElement('a'),
+            { target: '_blank', href: this._getSafeUrl(), rel: 'noopener noreferrer'}).click();
```

Comment:
Seems reasonable, although is there a reason not to put this link in directly rather than a button which makes a link and clicks on it?

Reply:
This is intended for consistency with the other menubar buttons which require JS onClick handlers. I can swap it out for this instance if you think that a simple link is better though?